askup/utils/general.py

```diff
@@ -121,10 +125,13 @@ def get_user_profile_rank_list(rank_user_id, viewer_user_id):
     Aquire and return a list of rows for the user profile rank list.
     """
     first_items = get_user_profile_rank_list_elements(viewer_user_id, args=(rank_user_id,))
-    result_row_datas, user_is_present = compose_user_profile_rank_list_row_data(first_items, rank_user_id)
+    result_row_datas, user_is_present = compose_user_profile_rank_list_row_data(
+        first_items, rank_user_id
+    )
+    total_users = get_rank_list_users_count()
 
     if user_is_present:
-        return result_row_datas
+        return result_row_datas, total_users
```

Comment:
Function description and name are not relevant to the returned object any more, need to be updated

Reply:
done